src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -274,66 +111,29 @@ class spell_pal_avenging_wrath : public SpellScriptLoader
         }
 };
 
-// 37877 - Blessing of Faith
```

Comment:
also still exists

Reply:
this spell is for item 30841 need to be moved to item_script or here is good?